AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -49,8 +49,12 @@ struct ConfirmCreatingSessionView: View {
             + Text(Strings.ConfirmCreatingSessionView.contentViewText_2)
             + Text(sessionName)
             .foregroundColor(.accentColor)
-            + Text(Strings.ConfirmCreatingSessionView.contentViewText_3)
-//            + Text(Strings.ConfirmCreatingSessionView.contentViewText_4)
+            + text
+    }
+    
+    private var text: Text {
```

Comment:
text is not a best name - there are plenty of texts here - it won't tell a reader anything and will force to look here to know -what- text is behind this name. I suggest you split this one up differently - using logical, easy to name chunks